ee/desktop/menu/menu_icon.go

```diff
@@ -0,0 +1,64 @@
+package menu
+
+import (
+	"runtime"
+
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/ee/ui/assets"
+)
+
+func (m *menu) getIcon(icon menuIcon) []byte {
+	switch icon {
+	case UnregisteredIcon:
```

Comment:
I think we should name these akin to what the icons are named. `Translucent`, `Default`, `TriangleExclaimation`, and `CircleX`